composites/Plugin/SnippetPreview/components/SnippetPreviewExample.js

```diff
@@ -0,0 +1,117 @@
+/* External dependencies */
+import React, { Component } from "react";
+import SnippetPreview from "./SnippetPreview";
+import styled from "styled-components";
+
+const Container = styled.div`
+	background-color: white;
+	margin: 5em auto 0; 
+`;
+
+export default class SnippetPreviewExample extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			title: "Welcome to the Gutenberg Editor - Local WordPress Dev. Snippet Title Snippet" +
+			" Title Snippet Title Snippet Title Snippet Title Snippet Title Snippet Title Snippet" +
+			" Title Snippet Title Snippet Title Snippet Title Snippet Title",
+			url: "https://local.wordpress.test/welcome-to-the-gutenberg-editor-2/",
+			description: "Of Mountains & Printing Presses The goal of this new editor is to make" +
+			" adding rich content to WordPress simple and enjoyable. This whole post is composed" +
+			" of. Of Mountains & Printing Presses The goal of this new editor is to make adding" +
+			" rich content to WordPress simple and enjoyable. This whole post is composed of. Of" +
+			" Mountains & Printing Presses The goal of this new editor is to make adding rich" +
+			" content to WordPress simple and enjoyable. This whole post is composed of Of Mountains" +
+			" & Printing Presses The goal of this new editor is to make adding rich content to" +
+			" WordPress simple and enjoyable. This whole post is composed of. Of Mountains & Printing" +
+			" Presses The goal of this new editor is to make adding rich content to WordPress" +
+			" simple and enjoyable. This whole post is composed of. Of Mountains & Printing Presses" +
+			" The goal of this new editor is to make adding rich content to WordPress simple and" +
+			" enjoyable. This whole post is composed of",
+			keyword: "editor",
+			date: "Jan 8, 2018",
+			locale: "en_US",
+			onClick( type ) {
+				// eslint-disable-next-line no-console
+				console.log( "clicked:", type );
+			},
+			hoveredField: "",
+			activeField: "description",
+			breadcrumbs: [ "hallo", "is", "it", "me", "you" ],
+		};
+
+		this.onMouseOver = this.onMouseOver.bind( this );
+		this.onMouseLeave = this.onMouseLeave.bind( this );
```

Comment:
might be good to define and set isAmp to true here, so people can see the AMP logo in the example.